agents/plugins/mk_docker.py

```diff
@@ -144,7 +144,7 @@ def get_config(cfg_file):
     files_read = config.read(cfg_file)
```

Comment:
Python agent  plugins are converted to python2 during build to support older python2-only hosts. Unfortunately `3to2` does not strip type annotations, so they have to remain comments. This goes for all files in `agents/plugins` :-(

Reply:
Python 2 died 1044 days ago on 1/1/2020.  Is it still supported?

Reply:
Everything under `agents/plugins` is meant to run on monitored hosts - we don't have any influence which (maybe deprecated) python version the customer is still using there. That's why we still provide a python 2.7 version. And as type annotations are not supported for python 2.x (https://peps.python.org/pep-0484/), we need to stick there with type comments.

Reply:
I am aware :-(
It is hard for us to tell what exactly is needed by how many users, as we have no telemetry of such things.
But there are industries that are running quite dated systems -- I wouldn't put it past them to have python2-only hosts.
Either way, dropping the support for that is something that product management will decide -- out of scope for a PR.
Needless to say that dropping the support would make things easier for us developers :-)

Reply:
Ok thanks - can you also squash the two commits in a single one?

Reply:
![Squash and Merge](https://user-images.githubusercontent.com/3709715/201073825-9db2d051-b3ab-4988-8d00-18b4967f2e45.png)